plugins/operation_history/include/golos/plugins/operation_history/history_object.hpp

```diff
@@ -0,0 +1,92 @@
+#pragma once
+
+#include <golos/protocol/authority.hpp>
+#include <golos/protocol/operations.hpp>
+#include <golos/protocol/steem_operations.hpp>
+
+#include <chainbase/chainbase.hpp>
+
+#include <golos/chain/index.hpp>
+#include <golos/chain/steem_object_types.hpp>
+
+#include <boost/multi_index/composite_key.hpp>
+
+//
+// Plugins should #define their SPACE_ID's so plugins with
+// conflicting SPACE_ID assignments can be compiled into the
+// same binary (by simply re-assigning some of the conflicting #defined
+// SPACE_ID's in a build script).
+//
+// Assignment of SPACE_ID's cannot be done at run-time because
+// various template automagic depends on them being known at compile
+// time.
+//
+
+#ifndef OPERATION_HISTORY_SPACE_ID
+#define OPERATION_HISTORY_SPACE_ID 14
+#endif
+
+
+namespace golos { namespace plugins { namespace operation_history {
+
+    enum account_object_types {
+        operation_object_type = (OPERATION_HISTORY_SPACE_ID << 8) + 1,
```

Comment:
why `+ 1` and why enum named `account_object_types ` if it uses different `SPACE_ID` ?
Looks like there is no `SPACE_ID` = 13.

Reply:
This is common rules for the code.
All indexes are initialized by the same rules.
I only keep code-style.